canvasapi/course.py

```diff
@@ -1410,17 +1410,15 @@ def get_multiple_submissions(self, **kwargs):
         :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
             :class:`canvasapi.submission.Submission`
         """
-        if 'grouped' in kwargs:
-            warnings.warn('The `grouped` parameter must be empty. Removing kwarg `grouped`.')
-            del kwargs['grouped']
+        cls = GroupedSubmission if 'grouped' in kwargs else Submission
```

Comment:
This will produce a bug if someone passed `grouped=False` to `get_multiple_submissions`.

Let's change this to:

```
cls = GroupedSubmission if kwargs.get('grouped') else Submission
```